src/bluetooth/device.hpp

```diff
@@ -99,6 +99,9 @@ class BluetoothDevice: public QObject {
 	Q_PROPERTY(bool batteryAvailable READ batteryAvailable NOTIFY batteryAvailableChanged);
 	/// Battery level of the connected device, from `0.0` to `1.0`. Only valid if @@batteryAvailable is true.
 	Q_PROPERTY(qreal battery READ default NOTIFY batteryChanged BINDABLE bindableBattery);
+	/// Received Signal Strength Indicator in dBm. Only valid when device is discoverable/advertising.
+	/// Typical values range from -30 (very close) to -90 (far away). Values of 0 indicate no signal data.
+	Q_PROPERTY(qint16 rssi READ rssi NOTIFY rssiChanged BINDABLE bindableRssi);
```

Comment:
If we're going to expose this at all, it should be normalized onto a linear scale that is easy to make a visual judgement with, and named something simpler, such as "signalStrength".